eng/common/scripts/Helpers/ApiView-Helpers.ps1

```diff
@@ -0,0 +1,64 @@
+function Map-LanguageToAPiview-Filter($language)
+{
+    $lang = $language
+    # Update language name to match those in API cosmos DB. Cosmos SQL is case sensitive and handling this within the query makes it slow.
+    if($lang -eq 'javascript'){
+        $lang = "JavaScript"
+    }
+    elseif ($lang -eq "dotnet"){
+        $lang = "C#"
+    }
+    elseif ($lang -eq "java"){
+        $lang = "Java"
+    }
+    elseif ($lang -eq "python"){
+        $lang = "Python"
+    }
+    return $lang
+}
+
+function Check-ApiReviewStatus($packageName, $packageVersion, $language)
+{
+  # Get API view URL and API Key to check status
+  Write-Host "Checking API review status"
+  try
+  {
+    az login -o none
+    $url = az keyvault secret show --name "APIURL" --vault-name "AzureSDKPrepRelease-KV" --query "value" --output "tsv"
```

Comment:
This should probably be a parameter that we pass in rather than making calls to KeyVault via the Azure CLI. The workflow will likely be run in Azure Pipelines at some point.

Reply:
I am a bit confused here. Prepare release script is triggered manually now and that is what going to be part of pipelines. right? I believe developer manually triggers prepare release script now so passing parameter to main script entry point is not possible without leaking api key. Another option is to get these keys fetched from KV by caller( Which is prepare release script) of this helper function.

Reply:
I'd say that the value itself can be a parameters to this function and we can move the gathering of it to the caller, in this case Prepare-Release script which a developer runs but in other cases we might want to pass it from our automation in devops.

Reply:
done